client/me/notification-settings/reader-subscriptions/index.jsx

```diff
@@ -243,21 +241,21 @@ class NotificationSubscriptions extends Component {
 
 						{ isAutomattician && (
 							<FormFieldset>
-								<FormLegend>
-									Disable auto-follow P2 posts upon commenting (Automatticians only)
-								</FormLegend>
+								<FormLegend>Auto-follow P2 posts (Automatticians only)</FormLegend>
 								<FormLabel>
 									<FormCheckbox
-										checked={ this.props.getSetting( 'p2_disable_autofollow_on_comment' ) }
+										checked={ ! this.props.getSetting( 'p2_disable_autofollow_on_comment' ) }
 										disabled={ this.props.getDisabledState() }
 										id="p2_disable_autofollow_on_comment"
 										name="p2_disable_autofollow_on_comment"
 										onChange={ this.props.toggleSetting }
-										onClick={ this.handleCheckboxEvent( 'Disable auto-follow P2 Upon Comment' ) }
+										onClick={ this.handleCheckboxEvent(
+											'Enable auto-follow P2 Upon Comment',
```

Comment:
Nit: let's apply case consistently:

```suggestion
											'Enable auto-follow P2 upon comment',
```